tests/raftstore/util.rs

```diff
@@ -391,16 +393,15 @@ pub fn must_read_on_peer<T: Simulator>(
     value: &[u8],
 ) {
     let timeout = Duration::from_secs(1);
-    match read_on_peer(cluster, peer, region, key, timeout) {
-        Ok(v) => if v != value {
-            panic!(
-                "read key {}, expect value {}, got {}",
-                escape(key),
-                escape(value),
-                escape(&v)
-            )
-        },
-        Err(e) => panic!("failed to read for key {}, err {:?}", escape(key), e),
+    let resp = read_on_peer(cluster, peer, region, key, false, timeout).unwrap();
```

Comment:
why not calling must_read_on_peer directly?

Reply:
It is in the `must_read_on_peer`.

Reply:
Better include the key.